src/Views/Home.test.jsx

```diff
@@ -1,6 +1,550 @@
-import { render, screen } from '@testing-library/react';
+import { fireEvent, render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import Home from './Home';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+
```

Comment:
Would prefer to see the house data imported from a separate data file. like from data/data.js